.gitignore

```diff
@@ -2,6 +2,9 @@
 # Created by https://www.gitignore.io/api/rust,jupyternotebooks
 # Edit at https://www.gitignore.io/?templates=rust,jupyternotebooks
 
+### VScode ###
+.vscode/
+
```

Comment:
see the top comment, .gitignore is generated. As gitignore.io is able to also generate code for vscode. Can you update gitignore with what is generated by https://www.gitignore.io/api/rust,jupyternotebooks,visualstudiocode

Reply:
done below